rs/replicated_state/src/canister_state/queues.rs

```diff
@@ -760,71 +740,65 @@ impl CanisterQueues {
         self.ingress_queue.count_bytes()
     }
 
-    /// Returns the number of canister messages enqueued in input queues.
+    /// Returns the number of non-stale canister messages enqueued in input queues.
     pub fn input_queues_message_count(&self) -> usize {
-        self.input_queues_stats.message_count
+        self.pool.message_stats().inbound_message_count
     }
 
     /// Returns the number of reserved slots across all input queues.
     ///
     /// Note that this is different from memory reservations for guaranteed
     /// responses.
     pub fn input_queues_reserved_slots(&self) -> usize {
-        self.input_queues_stats.reserved_slots as usize
-    }
-
-    /// Returns total amount of cycles included in input queues.
-    pub fn input_queue_cycles(&self) -> Cycles {
-        self.input_queues_stats.cycles
-    }
-
-    /// Returns the number of actual messages in output queues.
-    pub fn output_queues_message_count(&self) -> usize {
-        self.output_queues_stats.message_count
-    }
-
-    /// Returns total amount of cycles included in the output queues.
-    pub fn output_queue_cycles(&self) -> Cycles {
-        self.output_queues_stats.cycles
+        self.queue_stats.input_queues_reserved_slots
     }
 
     /// Returns the total byte size of canister input queues (queues +
     /// messages).
     pub fn input_queues_size_bytes(&self) -> usize {
-        self.input_queues_stats.size_bytes
+        self.pool.message_stats().inbound_size_bytes
+            + self.canister_queues.len() * CanisterQueue::EMPTY_SIZE_BYTES
```

Comment:
Nit: I'd just use the `size_of`.

Reply:
Changed.

I believe the constant was used in more than one place, at some point, so I wanted to make sure it's the same value everywhere. Right now, this was the only place where it was used, so `size_of()` is a lot clearer.